pallets/portfolio/src/benchmarking.rs

```diff
@@ -75,15 +55,15 @@ benchmarks! {
         let i in 1 .. 500;
         let mut items = Vec::with_capacity(i as usize);
         let target = UserBuilder::<T>::default().generate_did().build("target");
```

Comment:
This could be refactored as `user::<T>("target", 0)` using the convenience function from `benchs`.

Reply:
Let's do that in a different PR.